### ISSUE TITLE
editor.insertContent removes leading whitespace in webkit browsers

### ISSUE THREAD

**Issue by TinyMCE-User** [tinymce/tinymce#1720]
Description of problem:

Steps to reproduce:
1. Create a plugin which should do following:

onclick: function() {
  var text = editor.selection.getContent({ 'format' : 'text' });
  if (text && text.length > 0)
    editor.insertContent('a' + text + 'z')
}
1. Add this plugin to editor (obvious)
2. Type 'qwe awk qwe'
3. Select a word 'awk'
4. Press plugin button

Expected result: 'qwe aawkz qwe'

Actual result: 'qweaawkz qwe'

> _**Legacy information imported from TinyMCE bug tracker:**_
> 
> _[#T6992](http://archive.tinymce.com/develop/bugtracker_view.php?id=6992) posted by [babaika](http://archive.tinymce.com/profile.php?name=babaika)_
> 
> _**Tags:** []_
> _**Status:** Open_
> _**Resolution:** None_
> _**Attached URL:** none_

**Comment by mattcph** [comment]
I have found this with a plugin we have using insertContent().
In this case, insert an internal link anchor.

The specific fail appears on Safari/Chrome only after a double-click to select the text, then use plugin, call insertContent().

If I drag over text the issue does not appear.

The only odd thing I notice, Safari double click by default seems to select the trailing space as well after a  word. A slight drag back selects just the word, but perhaps this affects the carat position for the insertContent call.

**Comment by fyrkant** [comment]
Closing this due to old age, if anyone still has this issue in the latest version of TinyMCE please open up a new issue.
